### ISSUE TITLE
CI build for new package is failing because of various depexts problems

### ISSUE THREAD

**Issue by alavrik** [ocaml/opam-repository#12677]
CI build for this pull request used to pass successfully a week ago: https://github.com/ocaml/opam-repository/pull/12610

Now it is failing in various steps. All such failures seem to be related to depexts handling. For instance:
1. m4 is not installed: https://ci.ocamllabs.io/log/saved/docker-build-1fcc029d20e2c69ab27c5dc6d15b2bdc/e66283f738e7d8760d29ac01413c5ffcb11cd5c0
2. couldn't obtain source of opam-depext: https://travis-ci.org/ocaml/opam-repository/jobs/432061334
3. campl4-extra external dependency is missing: https://travis-ci.org/ocaml/opam-repository/jobs/432061332

Please advise on how to proceed. Any workarounds? Thanks!

**Comment by avsm** [comment]
This depext seems broken, as it hasn't added m4 to the list for some reason:

```
$ opam depext -uivyj 2 piqilib.0.6.14
# Detecting depexts using vars: arch=x86_64, os=linux, os-distribution=debian, os-family=debian
# No extra OS packages requirements found.
# All required OS packages found.
# Now letting OPAM install the packages
The following actions will be performed:
  - install ocamlbuild  0.12.0     [required by ulex, xmlm]
  - install dune        1.2.1      [required by jbuilder]
  - install conf-m4     1          [required by ocamlfind]
  - install camlp4      4.07+1     [required by ulex]
  - install jbuilder    transition [required by base64, easy-format]
  - install ocamlfind   1.8.0      [required by piqilib]
  - install result      1.3        [required by topkg]
  - install easy-format 1.3.1      [required by piqilib]
  - install base-bytes  base       [required by base64, ulex]
  - install topkg       0.9.1      [required by xmlm]
  - install ulex        1.2        [required by piqilib]
  - install base64      2.2.0      [required by piqilib]
  - install xmlm        1.3.0      [required by piqilib]
  - install piqilib     0.6.14*
===== 14 to install =====
```

**Comment by avsm** [comment]
Identified the issue and reported in https://github.com/ocaml/opam/issues/3557

The CI containers you are using actually used the `master` branch of opam. I'm now rebuilding them with the 2.0 branch, in which depext worked fine.  Once that finishes and deploys in a few hours, rerunning the CI should be a far happier experience for your packages. 

Sorry about the downtime!

**Comment by mseri** [comment]
This seems to be resolved now

**Comment by alavrik** [comment]
Thanks! @avsm @mseri how do I rerun the failed checks in #12610 ?

**Comment by avsm** [comment]
They should all have rerun by now with the fixed containers

**Comment by alavrik** [comment]
@avsm  reopen? -- I am still seeing 2 of the 3 original failures:

1. m4 command is not found: https://ci.ocamllabs.io/log/saved/docker-build-92e33c3d426a7e45db4c7b4f33bb2863/872f75e188722bcd376d6c7bddedcf982b0aa31f 

3. campl4-extra external dependency is missing: https://travis-ci.org/ocaml/opam-repository/jobs/432061332

also, can you explain how to trigger checks rerun? It is something that can be done only by repo maintainers? Thanks!

**Comment by rjbou** [comment]
[Fix](https://github.com/ocaml/opam/pull/3558) merged in opam master

**Comment by alavrik** [comment]
ping. Has these been resolved? If yes, how do I rerun the checks for #12610 ?

**Comment by alavrik** [comment]
I've just hit similar looking problem with camlp4-extra missing with a new pull request (https://github.com/ocaml/opam-repository/pull/13002): https://travis-ci.org/ocaml/opam-repository/jobs/456553616
